recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   skip: true  # [win or py<35]
-  number: 0
+  number: 1
```

Comment:
Not necessary because the first build didn't go in, but just to avoid confusion.